src/utils/publishToNpm.js

```diff
@@ -62,9 +62,17 @@ async function publishToNpm({
 
   if (await allowNpmPublish(version)) {
     await execWithOutput('npm', ['pack', '--dry-run'])
+
     if (opticToken) {
+      const packageInfo = await getPublishedInfo()
```

Comment:
can you make extra sure that:

- we don't call this function unnecessarily, e.g. if the information is already available to us
- in case we do need to retrieve this info, it is safe to do so in this place

Reply:
you're correct, I think we can just get it from the local package.json file here. Just updated the code.